### PR TITLE
Add default actions translation for list view

### PR DESCRIPTION
**Pull request by jordisala1991**
## Subject

I am targeting this branch, because this is BC.

## Changelog

```markdown
### Fixed
- Fix missing default translation for actions label on list view.
```

### CODE REVIEW

File: src/Resources/translations/SonataAdminBundle.fr.xliff [hunk 1]
```diff
@@ -446,6 +446,10 @@
         <source>link_actions</source>
         <target>Liste d'actions</target>
       </trans-unit>
+      <trans-unit id="list.label__actions">
+        <source>list.label__actions</source>
+        <target>Liste d'actions</target>
```
Comment (VincentLanglet) [inline src/Resources/translations/SonataAdminBundle.fr.xliff]: Liste d'actions would be translated as `Actions List` in english ; but we use `Actions` in english.
So I would use just
```suggestion
        <target>Actions</target>
```
here.

And I wonder if we shouldn't do the same for link_actions.

**Review by VincentLanglet — CHANGES_REQUESTED** [review]
I'm ok

File: src/Resources/translations/SonataAdminBundle.fr.xliff (hunk 1 above)
Comment (jordisala1991) [reply src/Resources/translations/SonataAdminBundle.fr.xliff]: Not sure, feel free to apply the modifications you consider, I just copied the link_actions.

**Comment by jordisala1991** [comment]
To me this will be useful mostly for our bundles, so we can only translate the default strategy, not all of them.

**Comment by VincentLanglet** [comment]
> To me this will be useful mostly for our bundles, so we can only translate the default strategy, not all of them.

Yeah I agree.
Also, I looked at others strategy and some of them should be deprecated IMHO.

**Comment by VincentLanglet** [comment]
> To me this will be useful mostly for our bundles, so we can only translate the default strategy, not all of them.

The default one is the native
https://github.com/sonata-project/SonataAdminBundle/blob/4.x/src/DependencyInjection/Compiler/AddDependencyCallsCompilerPass.php#L329

But we always use the underscore one
https://github.com/sonata-project/SonataPageBundle/blob/89013450f4e78cc79232d386ed27b9d0870c3030/src/Resources/config/admin.xml#L29

https://github.com/search?p=4&q=org%3Asonata-project+sonata.admin.label.strategy&type=Code

All the translator strategy:
- Noop https://github.com/sonata-project/SonataAdminBundle/blob/4.x/src/Translator/NoopLabelTranslatorStrategy.php
- Native https://github.com/sonata-project/SonataAdminBundle/blob/4.x/src/Translator/NativeLabelTranslatorStrategy.php
- Underscore https://github.com/sonata-project/SonataAdminBundle/blob/4.x/src/Translator/UnderscoreLabelTranslatorStrategy.php supported by your PR
- https://github.com/sonata-project/SonataAdminBundle/blob/4.x/src/Translator/FormLabelTranslatorStrategy.php 

https://github.com/sonata-project/SonataAdminBundle/blob/4.x/src/Translator/BCLabelTranslatorStrategy.php should be deprecated IMHO. Should we deprecate more translatorStrategy ?

File: src/Resources/translations/SonataAdminBundle.fr.xliff [hunk 2]
```diff
@@ -446,6 +446,10 @@
         <source>link_actions</source>
         <target>Liste d'actions</target>
```
Comment (VincentLanglet) [inline src/Resources/translations/SonataAdminBundle.fr.xliff]:
```suggestion
        <target>Actions</target>
```

**Review by VincentLanglet — APPROVED** [review]

**Comment by jordisala1991** [comment]
> > To me this will be useful mostly for our bundles, so we can only translate the default strategy, not all of them.
> 
> The default one is the native https://github.com/sonata-project/SonataAdminBundle/blob/4.x/src/DependencyInjection/Compiler/AddDependencyCallsCompilerPass.php#L329
> 
> But we always use the underscore one https://github.com/sonata-project/SonataPageBundle/blob/89013450f4e78cc79232d386ed27b9d0870c3030/src/Resources/config/admin.xml#L29
> 
> https://github.com/search?p=4&q=org%3Asonata-project+sonata.admin.label.strategy&type=Code
> 
> All the translator strategy:
> 
> * Noop https://github.com/sonata-project/SonataAdminBundle/blob/4.x/src/Translator/NoopLabelTranslatorStrategy.php
> * Native https://github.com/sonata-project/SonataAdminBundle/blob/4.x/src/Translator/NativeLabelTranslatorStrategy.php
> * Underscore https://github.com/sonata-project/SonataAdminBundle/blob/4.x/src/Translator/UnderscoreLabelTranslatorStrategy.php supported by your PR
> * https://github.com/sonata-project/SonataAdminBundle/blob/4.x/src/Translator/FormLabelTranslatorStrategy.php
> 
> https://github.com/sonata-project/SonataAdminBundle/blob/4.x/src/Translator/BCLabelTranslatorStrategy.php should be deprecated IMHO. Should we deprecate more translatorStrategy ?

I would say probably only the BCLabel IMO.

**Pull request merged**